hack/make/.build-rpm/docker-core.spec

```diff
@@ -0,0 +1,104 @@
+Name: docker-core
+Version: %{_rpmversion}
+Release: 1%{?dist}
+Summary: The open-source application container engine
+
+License: ASL 2.0
+Source: docker-core.tar.gz
+
+URL: https://dockerproject.com
+Vendor: Docker
+Packager: Docker <support@docker.com>
+
+BuildRequires: systemd
+Requires: /bin/sh
+Requires: libc.so.6
+Requires: libdevmapper.so.1.02
+Requires: libpthread.so.0
+Requires: libsqlite3.so.0
+Requires: iptables
+
+Conflicts: docker
+Conflicts: docker-io
+Conflicts: lxc-docker
+Conflicts: lxc-docker-virtual-package
```

Comment:
also, if there are conflicts that exist on one distro, but not the other; or version specific, you can do assertions like http://pkgs.fedoraproject.org/cgit/golang.git/tree/golang.spec#n51